pages/learn/develop/local-mode.md

```diff
@@ -122,14 +122,14 @@ These containers will have access to all the features and environment that balen
 
 ## SSH into the running app container or host OS
 
-If we want to run some test commands in our app container, we can do this easily using `{{ $names.company.short }} local ssh`. This command drops us directly into the selected container:
+If we want to run some test commands in our app container, we can do this easily using `{{ $names.company.short }} ssh`. This command drops us directly into the hostOS. From here, we can check system logs and [perform other troubleshooting tasks][troubleshooting]:
 ```
-sudo {{ $names.company.short }} local ssh f340127.local
+sudo {{ $names.company.short }} ssh f340127.local
 ```
-To connect to the host OS, we can add the `--host` option. From here, we can check system logs and [perform other troubleshooting tasks][troubleshooting]:
+To connect to a container, we can add the service name after the hostname.
```

Comment:
Suggestions: (feel free to disagree)
a container → an application container
hostname. → hostname: